flare/env.py

```diff
@@ -190,6 +200,97 @@ class to allow for njit acceleration with Numba.
     return bond_array_2, bond_positions_2, etypes
 
 
+@njit
+def get_2_body_arrays_ind(positions, atom: int, cell, cutoff_2: float, species):
```

Comment:
`get_2_body_arrays` and `get_2_body_arrays_ind` are redundant. The latter is used in the `get_m_body_arrays`, and should be changed into the similar style as `get_3_body_arrays`, i.e. search arrays from a shorter list (if we presume that 2-body cutoff is larger than many body)

Reply:
Yes, we did not have a deep knowledge of the code so we chose to duplicate the function to risk as little as possible and for lack of time,  sorry for this!

Reply:
No worry! The many-body code is very clean and well-organized overall.